deployer/templates/hawkular-metrics.yaml

```diff
@@ -19,6 +19,9 @@ parameters:
 - description: Specify the number of days that metrics should be stored for.
   name: METRIC_DURATION
   value: '7'
+- description: Internal URL for the master, for authentication retrieval
+  name: MASTER_URL
+  value: https://kubernetes.default.svc.cluster.local:443
```

Comment:
`cluster.local` suffix is configurable (although it isn't changed very often). Been advised to use just `kubernetes.default.svc` for master host if running in cluster.